README.md

```diff
@@ -142,15 +159,15 @@ abyme provides helper methods for all these. Here's how our form for `Project` l
   <%= f.input :description %>
   <%= f.submit 'Save' %>
 
-  <%= abymize(:tasks, f) do |abyme| %>
+  <%= f.abyme_for(:tasks) do |abyme| %>
     <%= abyme.records %>
     <%= abyme.new_records %>
     <%= add_association %>
   <% end %>
 <% end %>
 ```
 
-`abyme.records` will contain the persisted associations fields, while `abyme.new_records` will contain fields for the new associations. `add_association` will by default generate a button with a text of type "Add `resource_name`". To work properly, this method **has** to be called **inside the block** passed to the `abymize` method.
+`abyme.records` will contain the persisted associations fields, while `abyme.new_records` will contain fields for the new associations. `add_association` will by default generate a button with a text of type "Add `resource_name`". To work properly, this method **has** to be called **inside the block** passed to the `abyme_for` method.
```

Comment:
sorry wording again ^^ : unsure about `associations` 
which imo refers to the connection between two models 
wb: `associated records` or `child records` ?
👉 idem for `add_association`